.github/workflows/build_documentation.yml

```diff
@@ -0,0 +1,59 @@
+name: Build documentation
+
+on:
+  pull_request:
+    branches:
+      master
+  push:
+    branches:
+      master
+
+jobs:
+  docs:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        version:
+          - latest
+          - 0.2.5
+    steps:
+      - name: Set up python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ matrix.version == 'latest' && github.sha || matrix.version }}
+
+      - name: Fix-up requirement files for older tags
```

Comment:
We could drop documentation for 0.2.5 completely and enable docs for 0.2.6 once that is released. This would remove the need for this hacky workaround and we could use python 3.9 for all build steps. WDYT?